lib/Adapter/WorseReflection/Refactor/WorseGenerateAccessor.php

```diff
@@ -41,11 +41,16 @@ public function __construct(
         Reflector $reflector,
         Updater $updater,
         string $prefix = '',
-        bool $upperCaseFirst = false
+        bool $upperCaseFirst = null
     ) {
         $this->reflector = $reflector;
         $this->updater = $updater;
         $this->prefix = $prefix;
+        if ($prefix && null === $upperCaseFirst) {
+            $upperCaseFirst = true;
+        } elseif (null === $upperCaseFirst) {
+            $upperCaseFirst = false;
```

Comment:
would it make more sense to do `$this->upperCaseFirst = $prefix || $upperCaseFirst;`?

Reply:
Great! I also wanted to avoid this ugly if/elseif. Thanks!